net/i2pd/Makefile

```diff
@@ -36,45 +37,49 @@ define Package/i2pd
 endef
 
 define Package/i2pd/description
-	I2P (Invisible Internet Protocol) is a universal anonymous network layer. All
-	communications over I2P are anonymous and end-to-end encrypted, participants
-	don't reveal their real IP addresses.
-endef
-
-define Package/i2pd/conffiles
-/etc/config/i2pd
-/etc/i2pd/i2pd.conf
-/etc/i2pd/tunnels.conf
-/etc/i2pd/tunnels.d/
-endef
-
-define Build/Prepare
-	$(Build/Prepare/Default)
-	# race condition
-	cd $(PKG_BUILD_DIR); $(MAKE) mk_obj_dir
+  I2P (Invisible Internet Protocol) is a universal anonymous network layer. All
+  communications over I2P are anonymous and end-to-end encrypted, participants
+  don't reveal their real IP addresses.
 endef
 
 TARGET_LDFLAGS+=-latomic
 
 define Package/i2pd/install
 	$(INSTALL_DIR) $(1)/usr/sbin
 	$(INSTALL_BIN) $(PKG_BUILD_DIR)/i2pd $(1)/usr/sbin
+
+	# init.d script
+	$(INSTALL_DIR) $(1)/etc/init.d
+	$(INSTALL_BIN) ./files/i2pd.init $(1)/etc/init.d/i2pd
+
+	# certificates
 	$(INSTALL_DIR) $(1)/usr/share/i2pd
 	$(CP) $(PKG_BUILD_DIR)/contrib/certificates  $(1)/usr/share/i2pd
+
+	# service configuration
 	$(INSTALL_DIR) $(1)/etc/config
-	$(INSTALL_DATA) ./files/i2pd.config $(1)/etc/config/i2pd
+	$(INSTALL_CONF) ./files/i2pd.config $(1)/etc/config/i2pd
```

Comment:
`/etc/config` folder is for UCI configuration files. This was wrong even in past.

Reply:
So where I must save configuration for init.rc?

Reply:
That line installs UCI configuration btw, used by init script (net/i2pd/files/i2pd.config).

Reply:
Oh my god. I am so sorry! I didn't noticed that in the ``files`` folder, you have almost the same named files ``.conf`` and ``.config``.